pkg/cmd/server/api/v1/types.go

```diff
@@ -396,6 +396,8 @@ type MasterClients struct {
 
 // DNSConfig holds the necessary configuration options for DNS
 type DNSConfig struct {
+	// DNSDomain is the authoritative domain of the DNS server.
+	DNSDomain string `json:"dnsDomain"`
```

Comment:
Would prefer to call this field `ClusterDomain` since this is under the DNSConfig section and we may have other DNSDomains in the future.